### ISSUE TITLE
Error: "This browser or app may not be secure" asks to use different browser!!

### ISSUE THREAD

**Issue by MrLuxuri** [beakerbrowser/beaker#1912]
I recently moved from firefox to beaker browser and have been in the process of making it my daily browser but this really makes things hard for me. 
![Screen Shot 2021-02-26 at 2 19 28 PM](https://user-images.githubusercontent.com/71905307/109345270-deb9f880-783d-11eb-95ba-736f3998d829.png)
Please do fix this:)

**Comment by pfrazee** [comment]
Yeah this is a known issue and we're not sure how to get around it.

**Comment by MrLuxuri** [comment]
> Yeah this is a known issue and we're not sure how to get around it.
> […](#)

But google services is widely used and it is very important, users will need this to work properly to use the browser, what User Agent are you guys using? Maybe using Firefox, or chrome's user agent will work out?

**Comment by 00-000** [comment]
In 2019 Google caused the same problem for most independent browsers, and the Firefox user agent specific worked.

**Comment by fredhampton** [comment]
Has this issue been resolved ?

**Comment by paulkoeckdev** [comment]
## Quick Solution
```
app.userAgentFallback = "Chrome";
app.on("ready", ()=>{
    session.defaultSession.setUserAgent("Chrome");
}
```

[Source](https://stackoverflow.com/questions/59685927/electron-application-using-google-oauth-this-browser-or-app-may-not-be-secure)

## But
Google restricts Sign In's from non standard browsers for a couple of reasons!  A browser running on electron could for example simply have a keylogger installed. That's why google warns developers not to modify their browser's user-agent.

> The browser must identify itself clearly in the User-Agent when connecting to accounts.google.com. The browser must not use another browser's User-Agent string, such as Chrome or Firefox on that host. 

[Source](https://developers.googleblog.com/2020/08/guidance-for-our-effort-to-block-less-secure-browser-and-apps.html)
